harness/determined/common/util.py

```diff
@@ -127,3 +128,19 @@ def get_config_path() -> pathlib.Path:
         config_path = pathlib.Path.home().joinpath(".config")
 
     return config_path.joinpath("determined")
+
+
+def mask_config_dict(d: Dict) -> Dict:
```

Comment:
This isn't really a "utility" function at all... it only works on dictionaries which _happen_ to be the experiment config, which only really occurs in one place.

What I'm getting at is: in the absence of other places that we want to use this function, can we just put the function right in `harness.py`?

Reply:
Do you mean `launch_autohorovid.py` instead of `harness.py`?